src/models/thelio-r3-n1/repairs.md

```diff
@@ -0,0 +1,569 @@
+
+## Removing the CPU duct:
+
+The CPU duct guides airflow through the CPU cooler. It covers the CPU and partially obstructs the RAM slots and top GPU.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 7 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to remove the CPU duct:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case).
+2. Unplug both CPU fan connectors from the splitter board in the top corner of the machine.
+    - The splitter board provides the same signal to both ports, so it doesn't matter which fan is plugged into which port.
+
+![CPU fan connectors](./img/cpu-fan-connectors.webp)
+
+3. Unscrew the four back screws holding the CPU duct in place.
+
+![CPU duct screws (back)](./img/cpu-duct-screws-back.webp)
+![CPU duct screws (top)](./img/cpu-duct-screws-top.webp)
+
+4. Pull the CPU duct away from the machine.
+
+## Replacing the RAM:
+
+Thelio R3-N1 supports up to 64GB (2x32GB) of RAM. The RAM sticks are DDR5 DIMMs (non-ECC) running at a speed of up to 4800MHz. If you've purchased new RAM, need to replace your RAM, or are reseating your RAM, follow these steps.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 15 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the RAM:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case) and [remove the CPU duct](#removing-the-cpu-duct).
+    - The [side brace](#removing-the-side-brace) does not need to be removed, but it's been removed in the below photo for better visibility.
+2. To remove an existing RAM stick, flip the top latch up away from the stick, then pull the stick out of the slot, starting from the top edge. (The bottom of the RAM slot does not move.)
+
+![RAM slots](./img/ram-slots.webp)
+
+3. Make sure the tab on the top of the slot is open (pulled upwards), then insert the new RAM (or re-seat the existing RAM) into the slot, starting with the bottom edge.
+    - The RAM stick will only fit in one direction. The larger group of pins goes on top.
+    - If you are only using one RAM stick, place it in the right-side slot (farthest from the CPU.)
+4. Replace the top case.
+
+## Replacing the M.2 drives:
+
+Thelio R3-N1 has two M.2 slots (M key). Both slots are size 2280 and support PCIe NVMe Generation 4.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 30 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the M.2 drive (slot 1):
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case), [remove the side brace](#removing-the-side-brace), [remove the GPU](#replacing-the-gpu), and [remove the CPU duct](#removing-the-cpu-duct).
+    - Removing the CPU duct and GPU is optional but recommended to provide more working space and avoid damage.
+2. Unscrew and remove the M.2 heatsink.
```

Comment:
Not really, because you definitely don't need to unplug the heatsink to replace the drive. You only need to plug it in if you accidentally yank it out.

I guess I can mention that up here, though.